hyo2/ssm2/lib/atlas/regofsonline.py

```diff
@@ -368,7 +370,7 @@ def _build_opendap_url(self, input_date: dt, name: str) -> str:
                      name.lower(), input_date.strftime("%Y%m%d"))
         else:
             url = 'https://opendap.co-ops.nos.noaa.gov/thredds/dodsC/NOAA/%s/MODELS/%s/%s/%s/' \
-                  'nos.%s.regulargrid.n003.%s.t00z.nc' \
+                  '%s.t00z.%s.regulargrid.n003.nc' \
```

Comment:
fixed in https://github.com/hydroffice/hyo2_soundspeed/commit/051f58c051f77d98c7a002c350e50603b14f92a1